Dockerfile

```
@@ -1,6 +1,15 @@
-FROM alpine:3 
+FROM mcr.microsoft.com/dotnet/core/runtime:3.1
```

I wonder if we could use one of the already build images for alpine. They are way smaller and might help in keeping the runtime of these containers really quick.

https://github.com/dotnet/dotnet-docker/blob/master/samples/dotnetapp/README.md#build-an-image-for-alpine-debian-or-ubuntu

The samples use this line for thealpine container.  https://github.com/dotnet/dotnet-docker/blob/master/samples/dotnetapp/Dockerfile.alpine-x64

```suggestion
FROM mcr.microsoft.com/dotnet/core/runtime:3.1-alpine
```